src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,49 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class StringAddCalculator {
+    private final int NO_VALUE_RESPONSE = 0;
```

Comment:
```suggestion
    private static final int NO_VALUE_RESPONSE = 0;
```
관례상 상수로 만들어보면 어떨까요?

그리고 지금 상태라면 객체마다 고정되고 불필요한 `NO_VALUE_RESPONSE` 필드를 가지게 됩니다.
static으로 선언하면 전역으로 공유하는 클래스 변수가 되기 때문에 조금 더 나은 이점도 있어요

Reply:
그렇게 하는게 메모리를 아끼는 방법이 되겠네요. 그렇게 적용하겠습니다.